msg/AlarmInfo.msg

```diff
@@ -0,0 +1,25 @@
+# Alarm Number
+int32 number
+
+# Sub Code
+int32 sub_code
+
+# Alarm Name/Message
+string name
+
+# Contents
+string contents
+
+# Meaning
+string description
```

Comment:
These fields are direct copies of the names of the fields in the alarm documentation.

We can't actually retrieve these easily (see other comment).

We do have access to the "detailed text" (or sub code text).

Should we replace these fields by a `sub_code_message` field?

Reply:
Honestly, I have no idea what the `contents` and `description` are. 

I'm cool with just `number`, `sub_code`, and `name`